openshift/templates/services/audio/capture/deploy.yaml

```diff
@@ -0,0 +1,338 @@
+          containers:
+            - name: ${SOLUTION_NAME}-${APP_NAME}
+              image: ""
+              imagePullPolicy: Always
+              ports:
+                - containerPort: ${{APP_PORT}}
+                  protocol: TCP
+              resources:
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+              volumeMounts:
+                - name: ${SOLUTION_NAME}-${APP_NAME}
+                  mountPath: ${VOLUME_MOUNT_PATH}
+              env:
+                - name: KEYCLOAK_AUTH_SERVER_URL
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ${API_CONFIG_NAME}
+                      key: KEYCLOAK_AUTH_SERVER_URL
+                - name: KEYCLOAK_REALM
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ${API_CONFIG_NAME}
+                      key: KEYCLOAK_REALM
+                - name: KEYCLOAK_RESOURCE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ${API_CONFIG_NAME}
+                      key: KEYCLOAK_CLIENT_ID
+
+                - name: KAFKA_BOOTSTRAP_SERVERS
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ${API_CONFIG_NAME}
+                      key: KAFKA_BOOTSTRAP_SERVERS
+
+                - name: DB_URL
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ${API_CONFIG_NAME}
+                      key: DB_URL
+                - name: DB_USERNAME
+                  valueFrom:
+                    secretKeyRef:
+                      name: ${DB_SECRETS_NAME}
+                      key: POSTGRESQL_USER
+                - name: DB_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      name: ${DB_SECRETS_NAME}
+                      key: POSTGRESQL_PASSWORD
+
+                - name: KAFKA_CLIENT_ID
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ${SOLUTION_NAME}-${APP_NAME}
+                      key: KAFKA_CLIENT_ID
+                - name: KAFKA_LOGS_TOPIC
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ${SOLUTION_NAME}-${APP_NAME}
+                      key: KAFKA_LOGS_TOPIC
+                - name: MAX_FAILED_ATTEMPTS
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ${SOLUTION_NAME}-${APP_NAME}
+                      key: MAX_FAILED_ATTEMPTS
+                - name: DATA_SOURCE_MEDIA_TYPE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ${SOLUTION_NAME}-${APP_NAME}
+                      key: DATA_SOURCE_MEDIA_TYPE
+              # livenessProbe:
```

Comment:
These technically work but it takes so long for a Java Spring service to start that they will fail and cause the pod to not start.  I think I may configure to wait 5+ minutes just so they are running, but it somewhat defeats the purpose.